src/Command/AddUserCommand.php

```diff
@@ -199,7 +199,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $event = $stopwatch->stop('add-user-command');
         if ($output->isVerbose()) {
-            $this->io->comment(sprintf('New user database id: %d / Elapsed time: %.2f ms / Consumed memory: %.2f MB', $user->getId(), $event->getDuration(), $event->getMemory() / pow(1024, 2)));
+            $this->io->comment(sprintf('New user database id: %d / Elapsed time: %.2f ms / Consumed memory: %.2f MB', $user->getId(), $event->getDuration(), $event->getMemory() / (1024 ** 2)
+            ));
```

Comment:
this change looks suspicious to me

Reply:
You mean `pow(1024, 2)` -> `(1024 ** 2)` change?

Reply:
no, the CS change